todo_app/templates/index.html

```diff
@@ -7,12 +7,20 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+<div><form action='/add-item' method="POST">
+  <label for="title">add item</label><br>
+  <input type="text" id="item" name="item"><br><br>
```

Comment:
Minor, but we could add a `required` attribute to this input and we'd get some nice browser support preventing users from submitting an empty result (although without backend validation we've got no guarantees)